perfkitbenchmarker/benchmark_spec.py

```diff
@@ -159,6 +172,27 @@ def _GetOsTypeForGroup(self, group_name):
       return group_spec[OS_TYPE]
     return FLAGS.os_type
 
+  def _CheckBenchmarkSupport(self, cloud):
+    """ Check that the benchmark is supported with the given cloud provider. """
+
+    if FLAGS.benchmark_compatibility_checking is SKIP_CHECK:
+      True
```

Comment:
Not sure what the intent of this line is - "pass" is a no-op in python - "return" would also work